tests/nada-tests/src/variance.py

```diff
@@ -0,0 +1,16 @@
+from nada_dsl import *
+import nada_numpy as na
+
+def nada_main():
+    parties = na.parties(4)
+
+    a = na.array([4], parties[0], "A", SecretInteger)
```

Comment:
Nada arrays do not require re-initialization of each of the entries. `na.array` already does the initialization of the `SecretInteger`s within it.